src/testsuite/vec_perf_i128.c

```diff
@@ -124,4 +157,81 @@ int timed_mulluq (void)
   return rc;
 }
 
+#pragma GCC optimize ("no-unroll-loops")
```

Comment:
maybe a comment to explain why?  Is this particular to a certain level of the compiler?